db/docs/source/introduction.rst

```diff
@@ -62,6 +62,26 @@ The source imagery is linked via the ``capture_source_id`` attribute of building
 The NZ Aerial Surveys data dictionary is here: https://nz-imagery-surveys.readthedocs.io/en/latest/index.html
 
 
+Locality
+--------
+Locality includes suburb, locality, island and park reserve. Water features such as coastal bay, lake and inland bay are not included.
+
+For building outline which intersects with two localities, the locality with larger area of intersection is used for that feature.
+
+.. figure:: _static/locality_larger_area.png
+   :scale: 70%
+   :alt: Examples of locality
+
+   Image 3. Examples of building outline intersecting two localities.
+
+For building outline on water and not within any locality, the nearest locality is used for that feature.
+
+.. figure:: _static/locality_nearest.png
+   :scale: 70%
+   :alt: Examples of locality
+
+   Image 4. Examples of building outline not within any locality.
+
```

Comment:
This might be a little confusing at first. In general, the color blue is often used to represent water, but here you have it representing land, and water is white. Maybe try different colors, or use an actual example with the background image having a fade transparency, and the outline on top. One way would be to insersect our polygon layer (https://data.linz.govt.nz/layer/50293-nz-lake-polygons-topo-150k/) with the building outlines and pick those that are entirely inside the lake polygon. This might be an easy way to find examples (if we have any).

Reply:
I think I will just change colour because they are fake data so there is no aerial imagery...